controllers/handler.go

```diff
@@ -119,7 +122,9 @@ func (ech *EmqxClusterHandler) updateStatus(meta *cache.Meta) {
 
 // getLabels merges all the labels (dynamic and operator static ones).
 func (ech *EmqxClusterHandler) getLabels(emqx v1alpha2.Emqx) map[string]string {
-	dynLabels := map[string]string{}
+	dynLabels := map[string]string{
+		constants.LABEL_NAME_KEY: fmt.Sprintf("%s%c%s", emqx.GetNamespace(), '_', emqx.GetName()),
```

Comment:
Maybe `fmt.Sprintf("%s%c%s", emqx.GetNamespace(), '/', emqx.GetName()),` is better ? Because the name and namespace itself may contain "_"